net/sing-box/Makefile

```diff
@@ -119,9 +120,10 @@ GO_PKG_TAGS:=$(subst $(space),$(comma),$(strip \
 	$(if $(CONFIG_SINGBOX_WITH_WIREGUARD),with_wireguard) \
 ))
 
+
 define Package/sing-box/conffiles
-/etc/config/sing-box
-/etc/sing-box/
+	/etc/config/sing-box
```

Comment:
Why? What is the reason to do it?

Reply:
This is a mistake I made when formatting the code, sorry.